src/index.html

```diff
@@ -18,5 +18,60 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="image">
+        <img
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </div>
+
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="index.html"
```

Comment:
The `href` attribute for this link is set to `index.html`. If this is intended to navigate to different sections or pages, consider using unique URLs for each link.